corebehrt/classes/embeddings.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+
+
+# Constants (data dependent) for Time2Vec
+TIME2VEC_AGE_MULTIPLIER = 1e-2
+TIME2VEC_ABSPOS_MULTIPLIER = 1e-4
+TIME2VEC_MIN_CLIP = -100
+TIME2VEC_MAX_CLIP = 100
+
+
+class EhrEmbeddings(nn.Module):
+    """
+    Forward inputs:
+        input_ids: torch.LongTensor             - (batch_size, sequence_length)
+        token_type_ids: torch.LongTensor        - (batch_size, sequence_length)
+        position_ids: dict(str, torch.Tensor)   - (batch_size, sequence_length)
+            We abuse huggingface's standard position_ids to pass additional information (age, abspos)
+            This makes BertModel's forward method compatible with our EhrEmbeddings
+
+    Config:
+        vocab_size: int                         - size of the vocabulary
+        hidden_size: int                        - size of the hidden layer
+        type_vocab_size: int                    - size of max segments
+        layer_norm_eps: float                   - epsilon for layer normalization
+        hidden_dropout_prob: float              - dropout probability
+    """
+
+    def __init__(
+        self,
+        vocab_size: int,
+        hidden_size: int,
+        type_vocab_size: int,
+        layer_norm_eps: float,
+        hidden_dropout_prob: float,
+    ):
+        super().__init__()
+        self.LayerNorm = nn.LayerNorm(hidden_size, eps=layer_norm_eps)
+        self.dropout = nn.Dropout(hidden_dropout_prob)
+
+        # Initalize embeddings
+        self.concept_embeddings = nn.Embedding(vocab_size, hidden_size)
+        self.age_embeddings = Time2Vec(
+            1,
```

Comment:
This is always going to be `1` right? If so, perhaps we should just remove the `input_dim` argument for Time2Vec to simplify, as it doesn't make too much sense